src/main/java/com/spring/db/entity/user/User.java

```diff
@@ -0,0 +1,26 @@
+package com.spring.db.entity.user;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class User {
```

Comment:
Modify flow to save data into a table with name: `user` and fields: `id, first_name`, `last_name`, `middle_name` and `email`